data/remnant/remnant news.txt

```diff
@@ -0,0 +1,293 @@
+# Copyright (c) 2020 by Zitchas
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+
+news "remnant security advisory"
+	location
+		government "Remnant"
+	name
+		word
+			"System Security "
+			"Local "
+			"System "
+			"Defense "
+		word
+			"Advisory"
+			"Status"
+			"Threats"
+	message
+		word
+			"Incoming Korath Fleet in "
+		word
+			"00 days "
+			"01 days "
+			"03 days "
+			"04 days "
+			"05 days "
+			"06 days "
+			"07 days "
+		word
+			"0"
+			"1"
+		word
+			"0 hours. "
+			"1 hours. "
+			"2 hours. "
+			"3 hours. "
+			"4 hours. "
+			"5 hours. "
+			"6 hours. "
+			"7 hours. "
+			"8 hours. "
+			"9 hours. "
+			"A hours. "
+			"B hours. "
+			"C hours. "
+			"D hours. "
+			"E hours. "
+			"F hours. "
+		word
+			"Threat Sierra-"
+		word
+			"2"
+			"3"
+		word
+			"0"
+			"1"
+			"2"
+			"3"
+			"4"
+			"5"
+			"6"
+			"7"
+			"8"
+			"9"
+			"A"
+			"B"
+			"C"
+			"D"
+			"E"
+			"F"
```

Comment:
You could remove a lot of duplication by creating a phrase named "0 to F", then referencing that phrase whenever you want 0 to F to show up.
```suggestion
		phrase
			"0 to F"
```
And this would of course happen in multiple other places. The same could be done with the "00 to 07"